DbAdapterTestUtilities/Stubs/StubRecord.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#include "Mocks/MockRecord.h"
+#include "../Mocks/MockRecord.h"
```

Comment:
I would rather go the other way around:
`#include "DbAdapterTestUtilities/Mocks/MockRecord.h`

As that much should be valid when exporting the conan package and is much more clear where things are.

Reply:
Solved